### PR TITLE
Migrate AKassor from the legacy taxonomy

### PR DESCRIPTION
**Pull request by mokshasoft**

### CODE REVIEW

**Comment by weipe** [comment]
Looks good!

File: src/jobtech_taxonomy_database/types.clj [hunk 1]
```diff
@@ -37,6 +37,7 @@
 (def sun-education-level-1 "sun-education-level-1")
 (def sun-education-level-2 "sun-education-level-2")
 (def sun-education-level-3 "sun-education-level-3")
+(def unemployment-funds "unemployment-funds")
```
Comment (qeshi) [inline src/jobtech_taxonomy_database/types.clj:40]: Should we use singular for unemployment-funds?

Comment (weipe) [reply src/jobtech_taxonomy_database/types.clj:40]: We could ask David if they have some taxonomy conventions.

Comment (mokshasoft) [reply src/jobtech_taxonomy_database/types.clj:40]: Right, I forget that. The others were written in singular.

Comment (weipe) [reply src/jobtech_taxonomy_database/types.clj:40]: I asked an open question in the team-taxonomy chan

**Pull request merged**